test-fixtures/update.py

```diff
@@ -99,3 +99,4 @@ def largest_version():
             with open(tmp_file, 'w+') as tmpfile:
                 obj_yaml.dump(y, tmpfile)
                 copyfile(tmp_file, BITRISE_YML)
+                os.remove(tmp_file)
```

Comment:
I had to do that locally otherwise there was a tmp.yml file added to be commited. With this everything was working properly in my repo